butterfree/migrations/database_migration/database_migration.py

```diff
@@ -36,27 +41,40 @@ def _validate_schema(
             db_schema: object that contains the table og a given db schema.
 
         """
+        mismatches = []
```

Comment:
By just looking into the name of the method and its docstrings I could not understand what is this method about. What kind of validation is being done? It seems to me you are extracting the differences between db_schema and fs_schema. Isn't `_get_diff` a better name?

Reply:
Remember to change the typings and specifying what is this method returning.

Reply:
I've added the suggested changes.